cmd/stfinddevice/main.go

```diff
@@ -15,17 +16,19 @@ import (
 	"github.com/syncthing/syncthing/lib/discover"
 )
 
+const defServer = "https://announce.syncthing.net/?insecure&noannounce"
```

Comment:
It's out of date. Should be fixed, thanks. I think we should probably just default to one of the servers (`v4-1.discover.syncthing.net`?).